notebooks/useful_notebooks/01_find_and_release.ipynb

```diff
@@ -212,15 +215,45 @@
     "            msg = ('MATCHING ACCESS STATUS', a_type, raw_data['uuid'], raw_data['status'])\n",
     "        elif STATUS_LEVEL.get(raw_data['status']) == 0:\n",
     "            unusual += 1\n",
-    "            msg = ('SKIP UNUSUAL STATUS  ', a_type, raw_data['uuid'], raw_data['status'])\n",
+    "            msg = ('SKIP UNUSUAL STATUS   ', a_type, raw_data['uuid'], raw_data['status'])\n",
     "        else:\n",
     "            change += 1\n",
-    "            msg = ('       CHANGE        ', a_type, raw_data['uuid'], raw_data['status'])\n",
+    "            msg = ('        CHANGE        ', a_type, raw_data['uuid'], raw_data['status'])\n",
+    "            \n",
+    "            patch_data = {'status': change_status}\n",
+    "            \n",
+    "            # Special case: publication\n",
+    "            if change_status == 'released' and a_type in ['publication']:\n",
+    "                patch_data = {'status': 'current'}\n",
+    "            \n",
+    "            # Special case: HeLa cells (FASTQ and BAM files) are not released but restricted\n",
+    "            if STATUS_LEVEL[change_status] >=9 and a_type in ['file_fastq','file_processed'] and not ignore_hela_check:\n",
+    "                is_hela = False\n",
+    "                file_type = raw_data.get('file_format').get('file_format')\n",
+    "                if file_type in ['fastq', 'bam']:\n",
+    "                    if raw_data.get('experiments'):\n",
+    "                        for exp in raw_data['experiments']:\n",
+    "                            if 'HeLa' in exp['biosample']['biosource_summary']:\n",
+    "                                is_hela = True\n",
+    "                                break\n",
+    "                    elif raw_data.get('source_experiments'):\n",
+    "                        for source_exp in raw_data['source_experiments']:\n",
+    "                            exp = ff_utils.get_metadata(source_exp, key=my_auth)\n",
+    "                            if 'HeLa' in exp['biosample']['biosource_summary']:\n",
```

Comment:
let's change HeLa name check to lowercase, in case there are other way of writing it in the future
 if 'hela' in exp['biosample']['biosource_summary'].lower()

Reply:
```suggestion
    "                            if 'hela' in exp['biosample']['biosource_summary'].lower():\n",
```